spec/Section 2 -- Language.md

```diff
@@ -612,28 +612,32 @@ query noFragments {
 }
 ```
 
-The repeated fields could be extracted into a fragment and composed by a parent
-fragment or operation.
+The fields required by `friendProfile` can be extracted into a fragment and 
+composed by a parent fragment or operation.
 
 ```graphql example
 query withFragments {
   user(id: 4) {
     friends(first: 10) {
-      ...friendFields
+      ...friendProfileFragment
```

Comment:
I personally like `friendProfile`, `Fragment` is making me type more for no obvious reason as I know it's a fragment from the context.